cps/templates/index.xml

```diff
@@ -15,101 +15,101 @@
+    <link href="{{url_for('opds.feed_publisherindex')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_publisherindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by publisher')}}</content>
   </entry>
   <entry>
     <title>{{_('Categories')}}</title>
-    <link href="{{url_for('opds.feed_categoryindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_categoryindex')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_categoryindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by category')}}</content>
   </entry>
   <entry>
     <title>{{_('Series')}}</title>
-    <link href="{{url_for('opds.feed_seriesindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_seriesindex')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_seriesindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by series')}}</content>
   </entry>
   <entry>
     <title>{{_('Languages')}}</title>
-    <link href="{{url_for('opds.feed_languagesindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_languagesindex')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_languagesindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by Languages')}}</content>
   </entry>
   <entry>
     <title>{{_('Ratings')}}</title>
-    <link href="{{url_for('opds.feed_ratingindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_ratingindex')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_ratingindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by Rating')}}</content>
   </entry>
 
   <entry>
     <title>{{_('File formats')}}</title>
-    <link href="{{url_for('opds.feed_formatindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_formatindex')}}" type="application/atom+xml;profile=opds-catalog;kind=navigation"/>
     <id>{{url_for('opds.feed_formatindex')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Books ordered by file formats')}}</content>
   </entry>
   <entry>
     <title>{{_('Shelves')}}</title>
-    <link  href="{{url_for('opds.feed_shelfindex')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link  href="{{url_for('opds.feed_shelfindex')}}" type="application/atom+xml;profile=opds-catalog;kind=navigation"/>
     <id>{{url_for('opds.feed_shelfindex')}}</id>
```

Comment:
Calibre left this entry to it's feed and many opds reader are more compatible to calibre than to the spec and some of them are very strict, if there is anything different form what they are expecting it fails. Last time I changed something several readers caused problems -> No change